public/login.php

```diff
@@ -12,6 +12,7 @@
 
 if ($result->num_rows > 0) {
 	$_SESSION['user'] = $username;
+	$_SESSION['user_id'] = $result->fetch_assoc()['id'];
```

Comment:
It gets the database result as an associative array.

So if we got this back from the database:

```md
foo | bar | baz
-------------------
1 | 2 | 3
```